portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,82 @@
+function loadInFact(index, fact_index, isTruth) {
+    let fact = isTruth ? truths[fact_index] : lies[fact_index];
+
+    switch(index) {
+        case 0:
+            document.getElementById("first_txt").innerHTML = fact[0];
+            document.getElementById("first_img").src = fact[1];
+            break;
+        case 1:
+            document.getElementById("second_txt").innerHTML = fact[0];
+            document.getElementById("second_img").src = fact[1];
+            break;
+        case 2:
+            document.getElementById("third_txt").innerHTML = fact[0];
+            document.getElementById("third_img").src = fact[1];
+            break;
+    }
+}
+
+function loadInTruth() {
+
+    let first_truth_value = Math.floor(Math.random() * truths.length);
+    let second_truth_value = Math.floor(Math.random() * truths.length);
+    while (first_truth_value == second_truth_value) {
+        second_truth_value = Math.floor(Math.random() * truths.length);
+    }
+
+    let first_truth_index = Math.floor(Math.random() * 3);
+    let second_truth_index = Math.floor(Math.random() * 3);
+    while (first_truth_index == second_truth_index) {
+        second_truth_index = Math.floor(Math.random() * 3);
+    }
+
+    loadInFact(first_truth_index, first_truth_value, true);
+    loadInFact(second_truth_index, second_truth_value, true);
+
+    return [first_truth_index, second_truth_index];
+}
+
+
+function loadInLie(truth_indeces) {
```

Comment:
truthIndices  =)

(indices is spelled with two i's and one e)

Reply:
changed